src/main/kotlin/no/nav/familie/ba/sak/ekstern/pensjon/PensjonController.kt

```diff
@@ -88,6 +92,65 @@ class PensjonController(private val pensjonService: PensjonService) {
             )
         }
 
-        return ResponseEntity.ok(BarnetrygdTilPensjonResponse(pensjonService.hentBarnetrygd(request.ident, request.fraDato)))
+        return ResponseEntity.ok(
+            BarnetrygdTilPensjonResponse(
+                pensjonService.hentBarnetrygd(
+                    request.ident,
+                    request.fraDato,
+                ),
+            ),
+        )
+    }
+
+    @Operation(
+        description = "Tjeneste for Pensjon for å bestille identer med barnetrygd for et gitt år på kafka.",
+
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(
+                responseCode = "202",
+                description =
+                """
+                    RequestId som blir med kafka-meldingene
+                """,
+                content = [
+                    Content(
+                        mediaType = "application/json",
+                    ),
+                ],
+            ),
+            ApiResponse(
+                responseCode = "400",
+                description = "Ugyldig input. År må være av type string - 4 tegn",
+                content = [Content()],
+            ),
+            ApiResponse(
+                responseCode = "500",
+                description = "Uventet feil",
+                content = [
+                    Content(
+                        mediaType = "application/json",
+                    ),
+                ],
+            ),
+        ],
+    )
+    @GetMapping(
+        path = ["/bestill-personer-med-barnetrygd/{år}"],
+    )
+    fun bestillPersonerMedBarnetrygdForGittÅrPåKafka(
+        @PathVariable
+        år: String,
+    ): ResponseEntity<UUID> {
+        if (år.length != 4 && erTall(år)) {
```

Comment:
`år.length != 4` altså `<999` og `>9999`? 😛
Hva er det som gjør at vi ikke kan la jackson serialisere til tall? 

```suggestion
            fun bestillPersonerMedBarnetrygdForGittÅrPåKafka(
        @PathVariable
        år: Long,
    ): ResponseEntity<UUID> {
        return if (år in 1000..9999) {
            ResponseEntity.accepted().body(pensjonService.lagTaskForHentingAvIdenterTilPensjon(år))
        } else {
            throw IllegalArgumentException("$år er ikke et gyldig år")
        }
    }
```

Reply:
Oppdatert. La inn min/max-år (1900-2300) som strengere men fortsatt romslig validering.